es6/tests/utils.js

```diff
@@ -543,6 +543,15 @@ function createDoc(name) {
 	return loadDocument(name, documentCache[name].loadedContent);
 }
 
+function createDocV4(name, { modules }) {
+	const zip = documentCache[name].getZip();
+	return new Docxtemplater(zip, { modules });
+}
+
+function getZip(name) {
+	return documentCache[name].getZip();
```

Comment:
Can you instead do : `return new Pizzip(documentCache[name].loadedContent)`

Reply:
Does `getZip` have performance issues? Isn't the documents cached when tests start running?

Reply:
I prefer to do `new Pizzip(documentCache[name].loadedContent)` because when you call `.getZip`, you get a zip file that could have been modified by some other test.

with `new Pizzip(documentCache[name].loadedContent)` you are sure that you get the same content even if the zip was already used in an other test.